lib/help/english.js

```diff
@@ -54,7 +54,9 @@ export default {
 **\`!wolfram\`** \`query\`
 		Query Wolfram Alpha for almost anything
 **\`!videocall\`**
-    Start a one click video call or screenshare directly on Appear.in. Mention users to make it private.`,
+    Start a one click video call or screenshare directly on Appear.in. Mention users to make it private.
+**\`!paste\`**\`text\`
```

Comment:
Were trying to keep the help section alphabetical. Mind moving it up a bit, please? :)